docs/react/nextjs.md

```diff
@@ -0,0 +1,73 @@
+# Next.js
+
+## SSR
+Highcharts does not support server environments. In order to use it within recent versions (v13+)
```

Comment:
"Highcharts does not support server environments" - maybe rather something like:

"Highcharts interactive charts can not be generated on the server side, as the interactivity relies on a client environment. Static images can be generated using the Highcharts Export Server (docs link to https://www.highcharts.com/docs/export-module/setting-up-the-server).

In order to use Highcharts interactive charts within recent versions..."

?